files/celery_beat_start

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+NAME="amphora"                                  # Name of the application
+DJANGODIR=/var/opt/wsgi/amphora/amphora             # Django project directory
+SOCKFILE=/var/opt/wsgi/amphora/run/amphora.sock  # we will communicte using this unix socket
+USER=amphora                                        # the user to run as
+GROUP=webapps                                     # the group to run as
+NUM_WORKERS=3                                     # how many worker processes should Gunicorn spawn
+DJANGO_SETTINGS_MODULE=jars.settings             # which settings file should Django use
+DJANGO_WSGI_MODULE=jars.wsgi                     # WSGI module name
+DJANGO_CELERY_MODULE=jars.celery                     # WSGI module name
```

Comment:
let's remove the ones that are not used

Reply:
and let's adjust the NAME variable to accurately reflect which process is started